### PR TITLE
tiflash: add tiflash configuration document

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Add TiFlash configuration document.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/2603
- Other reference link(s):

### CODE REVIEW

File: reference/tiflash/configuration.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+path = The TiFlash data storage path.     # If there are multiple directories, separate each directory with a comma. For example, `/ssd_a/data/tiflash,/hdd_b/data/tiflash,/hdd_c/data/tiflash`. If your environment has multiple disks, it is recommended that each path corresponds to one disk and you put disks with best performance at the front to fully use performance of all disks.
+path_realtime_mode = false # The default value is `false`. If you set it to `true` and multiple directories are deployed in the path, the latest data is stored in the first directory and older data is stored in the rest directories.
+listen_host = The TiFlash service listening host. # Generally, it is configured as 0.0.0.0.
+tcp_port = The TiFlash TCP service port.
+http_port = The TiFlash HTTP service port.
+```
+
+```
+[flash]
+    tidb_status_addr = TiDB status address and port. # Multiple addressed are separated with commas.
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply reference/tiflash/configuration.md:40]: port and address

**Comment by TomShawn** [comment]
@ran-huang PTAL again, thanks!

File: reference/tiflash/configuration.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+This section introduces the configuration parameters of TiFlash.
+
+### Configure the `tiflash.toml` file
+
+```
+tmp_path = The path in which the TiFlash temporary files are stored.
+path = The TiFlash data storage path.     # If there are multiple directories, separate each directory with a comma. For example, `/ssd_a/data/tiflash,/hdd_b/data/tiflash,/hdd_c/data/tiflash`. If your environment has multiple disks, it is recommended that each path corresponds to one disk and you put disks with best performance at the front to fully use performance of all disks.
+path_realtime_mode = false # The default value is `false`. If you set it to `true` and multiple directories are deployed in the path, the latest data is stored in the first directory and older data is stored in the rest directories.
+listen_host = The TiFlash service listening host. # Generally, it is configured as 0.0.0.0.
```
Comment (ran-huang) [inline reference/tiflash/configuration.md:33]:
```suggestion
listen_host = The TiFlash service listening host. # Generally, it is configured as `0.0.0.0`.
```

File: reference/tiflash/configuration.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+http_port = The TiFlash HTTP service port.
+```
+
+```
+[flash]
+    tidb_status_addr = TiDB status port and address. # Multiple addresses are separated with commas.
+    service_addr = The listening address of TiFlash Raft services and coprocessor services.
+```
+
+Multiple TiFlash nodes elect a master to add or delete placement rules to PD and, you need three parameters to control this process.
```
Comment (ran-huang) [inline reference/tiflash/configuration.md:44]:
```suggestion
Multiple TiFlash nodes elect a master to add or delete placement rules to PD, and you need three parameters to control this process.
```

**Pull request merged**